src/pages/rest/modules/import/index.md

```diff
@@ -52,6 +52,10 @@ POST /rest/<store_view_code>/V1/import/csv
 |`validationStrategy` | Strategy to use when entities are invalid | string | Required. Either `validation_stop_on_errors` or `validation-skip-errors` |
 |`allowedErrorCount` | The maximum number of errors that can occur until import is canceled | string | Required |
 |`csvData` | Base64 encoded string containing CSV data | string | Required |
+|`ImportFieldSeparator` | Separator used in CSV.  Default is ',' | string | Optional |
+|`ImportMultipleValueSeparator` | Separator used in columns with multiple values such as categories.  Default is ',' | string | Optional |
+|`ImportEmptyAttributeValueConstant` | Constant to be replaced with empty attribute | string | Optional |
```

Comment:
FYI,  this option in Web Admin is missing documentation.  When searching for documentation on it,  I found comments on forums that suggest that it May have been a feature added in Magento 2.3.  

I wanted to add a link to the existing documentation for this option in web admin interface's importer,  but because it doesn't exist,  I couldn't.  It's a lot more complicated to explain than the other options and deserves examples if we expect people to understand how to use this.

Reply:
Thanks for the updates @JacobBrownAustin . I knew this topic looked familiar. I processed the original contribution in #28 .

I have a couple of favors to ask:

* Would you add a Head1 before line 6 called **ImportCsv API** (and also a blank line below that)?
* Either go to https://experienceleague.adobe.com/docs/commerce-admin/systems/data-transfer/import/data-import.html?lang=en and create an issue, providing any info you have, or create a PR to update the topic.

Reply:
@keharper 
* I'm not sure what you mean by Head1?
* Created https://github.com/AdobeDocs/commerce-admin.en/issues/23

Reply:
Ah, I see that was fixed in #71